app/src/main/java/com/github/ybecker/epforuml/latex/MathView.kt

```diff
@@ -0,0 +1,187 @@
+package com.github.ybecker.epforuml.latex
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.util.AttributeSet
+import android.util.Log
+import android.view.MotionEvent
+import android.webkit.WebView
+import androidx.core.content.ContextCompat
+import com.github.ybecker.epforuml.R
+
+/**
+ * Created by lingaraj on 3/15/17.
+ * The katex.hourglass.in.mathlib.MathView library directly imported to the project
+ * to be able to modify the html code to display the latex.
+ */
+class MathView : WebView {
+    private val TAG = "KhanAcademyKatexView"
+    private var display_text: String? = null
+    private var text_color = 0
+    private var text_size = 0
+    private var clickable = false
+    private var enable_zoom_in_controls = false
+
+    constructor(context: Context) : super(context) {
+        configurationSettingWebView(enable_zoom_in_controls)
+        setDefaultTextColor(context)
+        setDefaultTextSize()
+    }
+
+    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs) {
+        configurationSettingWebView(enable_zoom_in_controls)
+        val mTypeArray = context.theme.obtainStyledAttributes(
+            attrs,
+            R.styleable.MathView,
+            0, 0
+        )
+        try {
+            setBackgroundColor(
```

Comment:
could refactor this whole inner try block

Reply:
I forgot to mention the MathView library comes from another project on git